Just restarted it, but the build failure looks real (probably just need to check for `None` and not add it)
```
======================================================================
ERROR: test_multiple_children (custom.opm.tests.test_child_condition_four.TestChildBirthRegistration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/custom/opm/tests/test_child_condition_four.py", line 80, in test_multiple_children
    row = MockCaseRow(case, self.report, child_index=2)
  File "/mnt/custom/opm/tests/case_reports.py", line 118, in __init__
    super(MockCaseRow, self).__init__(case, report, child_index=child_index, **kwargs)
  File "/mnt/custom/opm/beneficiary.py", line 78, in __init__
    self.set_case_properties()
  File "/mnt/custom/opm/beneficiary.py", line 270, in set_case_properties
    color="red" if self.cash_amt == 0 else "green",
  File "/mnt/custom/opm/beneficiary.py", line 741, in cash_amt
    return self.month_amt + self.year_end_bonus_cash
TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
```

--- a/custom/opm/beneficiary.py
+++ b/custom/opm/beneficiary.py
@@ -735,6 +735,8 @@ class OPMCaseRow(object):
                 2: TWO_YEAR_AMT,
                 3: THREE_YEAR_AMT,
             }.get(year_value, 0)
+        else:
+            return 0
 
     @property
     def cash_amt(self):

